lib/fixed_average_info.rb

```diff
@@ -0,0 +1,61 @@
+class FixedAverageInfo
```

Comment:
It's kinda based on `lib/AverageInfo.rb`, the surprising part is that this other class filters for tags, usernames and so on. I didn't add these here cause I think it's obsolete. Since we are getting sessions ids, and these ids come from the frontend. And the frontend fetched based on these filters already. 
Unless there is something I'm no aware of?